sdk/metricsadvisor/Azure.AI.MetricsAdvisor/tests/MetricsAdvisorClient/MetricFeedbackLiveTests.cs

```diff
@@ -0,0 +1,290 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using Azure.AI.MetricsAdvisor.Models;
+using Azure.Core.TestFramework;
+using NUnit.Framework;
+
+namespace Azure.AI.MetricsAdvisor.Tests
+{
+    public class MetricFeedbackLiveTests : MetricsAdvisorLiveTestBase
+    {
+        public MetricFeedbackLiveTests(bool isAsync) : base(isAsync)
+        {
+        }
+
+        private DateTimeOffset CreatedFeedbackStartTime => DateTimeOffset.Parse("2020-09-26T00:00:00Z");
+
+        private DateTimeOffset CreatedFeedbackEndTime => DateTimeOffset.Parse("2020-09-29T00:00:00Z");
+
+        private DateTimeOffset FeedbackSamplingStartTime => DateTimeOffset.Parse("2020-12-01T00:00:00Z");
+
+        private DateTimeOffset FeedbackSamplingEndTime => DateTimeOffset.Parse("2020-12-31T00:00:00Z");
+
+        [RecordedTest]
+        [TestCase(true)]
+        [TestCase(false)]
+        public async Task AddAndGetAnomalyFeedback(bool populateOptionalMembers)
+        {
+            MetricsAdvisorClient client = GetMetricsAdvisorClient();
+
+            var dimensionKey = new DimensionKey();
+            dimensionKey.AddDimensionColumn("city", "Delhi");
+            dimensionKey.AddDimensionColumn("category", "Handmade");
+
+            var filter = new FeedbackDimensionFilter(dimensionKey);
+
+            var feedbackToAdd = new MetricAnomalyFeedback(MetricId, filter, CreatedFeedbackStartTime, CreatedFeedbackEndTime, AnomalyValue.AutoDetect);
+
+            if (populateOptionalMembers)
+            {
+                feedbackToAdd.AnomalyDetectionConfigurationId = DetectionConfigurationId;
+            }
+
+            string feedbackId = await client.AddFeedbackAsync(feedbackToAdd);
+
+            Assert.That(feedbackId, Is.Not.Null);
+
+            MetricFeedback addedFeedback = await client.GetFeedbackAsync(feedbackId);
+
+            ValidateMetricFeedback(addedFeedback, feedbackId, dimensionKey);
+
+            Assert.That(addedFeedback.Type, Is.EqualTo(FeedbackType.Anomaly));
+
+            var anomalyFeedback = addedFeedback as MetricAnomalyFeedback;
+
+            Assert.That(anomalyFeedback, Is.Not.Null);
+            Assert.That(anomalyFeedback.AnomalyValue, Is.EqualTo(AnomalyValue.AutoDetect));
+            Assert.That(anomalyFeedback.StartTime, Is.EqualTo(CreatedFeedbackStartTime));
+            Assert.That(anomalyFeedback.EndTime, Is.EqualTo(CreatedFeedbackEndTime));
+
+            if (populateOptionalMembers)
+            {
+                Assert.That(anomalyFeedback.AnomalyDetectionConfigurationId, Is.EqualTo(DetectionConfigurationId));
+                // TODO: Add snapshot validation (https://github.com/azure/azure-sdk-for-net/issues/15915).
```

Comment:
Left snapshot validation for later, since it will be pretty much the same as validating a Detection Configuration (and we intend to add tests for that as part of #15915).